src/test/java/com/puppycrawl/tools/checkstyle/AbstractGuiTestSupport.java

```diff
@@ -48,8 +48,9 @@ public static void validateGraphicsEnvironment() {
      * @param name the name of component to find
      * @param <T> the type of component to find
      * @return the component if found, {@code null} otherwise
-     * @noinspection unchecked
+     * @noinspection unchecked, ParameterNameDiffersFromOverriddenParameter
      * @noinspectionreason unchecked - we know that any component is OK to typecast to T
+     * @noinspectionreason ParameterNameDiffersFromOverriddenParameter - false positive
```

Comment:
False positive is from inspections mistaking enhanced for loop variable for method parameter.

Reply:
I do not think this inspection is mature enough, not much configurations to ease enforcing to be same.
usually libraries or base classes do not have context of how they are used, so names in them are general, but it is good to name more context specific names in child classes and follow they own terminology.

if we can not put on leash this inspection rule, I would recommend to disable it at all.